lib/tasks/scripts/telemedicine_reports.rb

```diff
@@ -0,0 +1,322 @@
+# frozen_string_literal: true
+
+require "csv"
+
+module TelemedicineReports
+  class << self
+    def parse_mixpanel(mixpanel_csv_path)
+      mixpanel_csv = File.read(mixpanel_csv_path)
+      mixpanel_data = CSV.parse(mixpanel_csv, headers: false)
+
+      mixpanel_data.drop(1).map { |row|
+        facility = User.find(row[1]).registration_facility
+        {user_id: row[1],
+         date: Date.parse(row[2]),
+         clicks: row[3].to_i,
+         facility_id: facility.id,
+         district: facility.district,
+         state: facility.state,
+         type: facility.facility_type}
+      }
+    end
+
+    def generate_report(mixpanel_data, period_start, period_end)
```

Comment:
Is this supposed to be used like `TelemedicineReports.generate_report(TelemedicineReports.parse_mixpanel('path'), ...)`?

Reply:
There's a usage here: https://github.com/simpledotorg/simple-server/blob/242176467e371b39261c88f21f8761573dde6bc0/lib/tasks/reports.rake#L24

But, essentially yes